replay_parser/reader.py

```diff
@@ -27,29 +35,29 @@ def read_string(self) -> str:
         """
         Parses string from binary data.
         """
-        res = b""
+        res = bytearray()
         while True:
-            char_ = unpack("s", self.buffer.read(1))[0]
+            char_ = unpack("c", self.buffer.read(1))[0]
```

Comment:
maybe if you're putting everything in little endian, we should use this `<` ?

Reply:
Strings don't have endianness. And endianness doesn't make sense for single byte objects such as `byte` or `char`

Reply:
ok, then ignore